quadlet/bluesky_pds/caddy.container

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Caddy reverse proxy Quadlet
+After=network-online.target
+Wants=network-online.target
```

Comment:
No need for these dependencies. Quadlet will add them automatically. Furthermore, it will set the correct dependencies based on whether the unit is rootful or rootless